sysutils/shmux/Portfile

```diff
@@ -3,26 +3,26 @@
 PortSystem          1.0
 
 name                shmux
-version             1.0.2
+version             1.0.3
 revision            1
```

Comment:
sorry, one more thing: when updating the version you should set the revision back to zero

Reply:
Done, thanks for the review!